src/main/java/net/minestom/server/instance/generator/GenerationUnit.java

```diff
@@ -0,0 +1,25 @@
+package net.minestom.server.instance.generator;
+
+import net.minestom.server.coordinate.Point;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public interface GenerationUnit {
+    @NotNull UnitModifier modifier();
+
+    @NotNull Point size();
+
+    @NotNull Point absoluteStart();
+
+    @NotNull Point absoluteEnd();
```

Comment:
Well `absolute` is the redundant part since a generation unit can't be relative because we have no reference point, however the type of the point (section/block) is more important, and as I said it could be confusing at first. But if you feel like it, just mark it as resolved.

Reply:
After a small internal vote, we decided to keep the current methods